Views/ldb_clicks.view.lkml

```diff
@@ -0,0 +1,70 @@
+view: ldb_clicks {
+  label: "LDB Clicks"
+  derived_table: {
+    sql: SELECT
+          lc.root_id AS root_id,
+          wp.id AS page_view_id,domain_sessionid AS session_id,
+          COALESCE(events.page_urlhost,'') AS page_urlhost,
+          events.page_url,
+          click_type,
+          lc.sku,
+          CASE WHEN click_type = 'add_to_cellar' THEN 1 ELSE 0 end AS add_to_cellar_count,
+          CASE WHEN click_type = 'where_to_buy' THEN 1 ELSE 0 end AS where_to_buy_count,
+          CONVERT_TIMEZONE('UTC', 'America/Vancouver', lc.root_tstamp) AS timestamp
+
+        FROM atomic.ca_bc_gov_ldb_click_1 AS lc
+          LEFT JOIN atomic.com_snowplowanalytics_snowplow_web_page_1 AS wp
+              ON lc.root_id = wp.root_id AND lc.root_tstamp = wp.root_tstamp
+          LEFT JOIN atomic.events ON lc.root_id = events.event_id AND lc.root_tstamp = events.collector_tstamp
+          LEFT JOIN microservice.ldb_sku ON ldb_sku.sku = lc.sku
+        WHERE {% incrementcondition %} timestamp {% endincrementcondition %} -- this matches the table column used by increment_key
+        ;;
+    distribution_style: all
```

Comment:
Not a change request, just a comment on how Redshift manages distribution style:

> Redshift assigns an optimal distribution style based on the size of the table data... it initially assigns ALL distribution to a small table, then changes to EVEN distribution when the table grows larger. 

details: https://docs.aws.amazon.com/redshift/latest/dg/c_choosing_dist_sort.html

So setting the distribution style to `ALL` is what Redshift would probably do for this table anyway, unless Redshift itself determined `EVEN` were better, based on the table size.